pkg/subctl/cmd/join.go

```diff
@@ -80,6 +81,8 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&submarinerDebug, "subm-debug", false, "Enable Submariner debugging (verbose logging)")
 	cmd.Flags().BoolVar(&noLabel, "no-label", false, "skip gateway labeling")
 	cmd.Flags().StringVar(&brokerClusterContext, "broker-cluster-context", "", "Broker cluster context")
+	cmd.Flags().StringVar(&cableDriver, "cable-driver", "ipsec/strongswan", "cable[/driver] - Cable type and optional driver implementation")
```

Comment:
I think we should specify "" default value and let the engine backend decide what default driver to use. That way if we change the default driver in the backend in a future release, we don't have to change it here and upgrades will automatically pick up the new default.

Reply:
Done